src/dataset.rs

```diff
@@ -1,18 +1,50 @@
 use crate::commands::InitMissionArgs;
 use crate::manifest::Manifest;
-use chrono::serde::ts_seconds;
-use chrono::DateTime;
-use chrono::Utc;
-use serde::{Deserialize, Serialize};
+use anyhow::Result;
+use chrono::{DateTime, FixedOffset, SecondsFormat};
+use serde::de;
+use serde::de::{Deserializer, Visitor};
+use serde::{Deserialize, Serialize, Serializer};
 use std::collections::{HashMap, HashSet};
 use std::path::PathBuf;
-use std::vec;
+use std::{fmt, vec};
+
+fn serialize_timestamp<S>(date: &DateTime<FixedOffset>, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    let s = date.to_rfc3339_opts(SecondsFormat::Secs, true);
+    serializer.serialize_str(&s)
+}
+
+fn deserialize_timestamp<'de, D>(deserializer: D) -> Result<DateTime<FixedOffset>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    struct DateTimeVisitor;
+
+    impl<'de> Visitor<'de> for DateTimeVisitor {
+        type Value = DateTime<FixedOffset>;
+
+        fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+            formatter.write_str("RFC3339 formatted string")
+        }
+
+        fn visit_str<E>(self, value: &str) -> Result<Self::Value, E>
+        where
+            E: de::Error,
+        {
+            DateTime::parse_from_rfc3339(value).map_err(de::Error::custom)
+        }
+    }
+
+    deserializer.deserialize_str(DateTimeVisitor)
+}
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Dataset {
     root: std::path::PathBuf,
-    #[serde(with = "ts_seconds")]
-    day_0: DateTime<Utc>,
+    day_0: DateTime<FixedOffset>,
```

Comment:
Probably something I'm missing, any particular reason why we don't need to specify serialize and deserialize functions here?

Reply:
Added to latest commit. The built in serde function didn't follow the standard we were using in the examples, so we needed to do our own serialize/deserialize functions.